docs/en/operations/system-tables/server_settings.md

```diff
@@ -0,0 +1,52 @@
+---
+slug: /en/operations/system-tables/server_settings
+---
+# server_settings
+
+Contains information about global settings for the server, which were specified in `config.xml`.
+Currently, the table shows only settings from the first layer of `config.xml` and doesn't support nested configs (e.g. [logger](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-logger)).
+
+Columns:
+
+-   `name` ([String](../../sql-reference/data-types/string.md)) — Server setting name.
+-   `value` ([String](../../sql-reference/data-types/string.md)) — Server setting value.
+-   `default` ([String](../../sql-reference/data-types/string.md)) — Server setting default value.
+-   `changed` ([UInt8](../../sql-reference/data-types/int-uint.md#uint-ranges)) — Shows whether a setting was specified in `config.xml`
+-   `description` ([String](../../sql-reference/data-types/string.md)) — Short server setting description.
+-   `type` ([String](../../sql-reference/data-types/string.md)) — Server setting value type.
+
+**Example**
+
+The following example shows how to get information about server settings which name contains `thread_pool`.
+
+``` sql
+SELECT *
+FROM system.server_settings
+WHERE name LIKE '%thread_pool%'
+```
+
+``` text
+┌─name─────────────────────────┬─value─┬─default─┬─changed─┬─description───────────────────────────┬─type───┐
+│ max_thread_pool_size         │ 10000 │ 10000   │       0 │ Max size for global thread pool.      │ UInt64 │
```

Comment:
Guess that for illustration purposes, the table should also include changed settings.